packages/common/src/PropertyExt.ts

```diff
@@ -682,10 +682,21 @@ export class PropertyExt extends Class {
     }
 
     hashSum(ignore: string[] = [], more = {}): string {
-        ignore = [...ignore, "fields", "classed"];
+        ignore = [...ignore, "classed"];
         const props: { [key: string]: any } = more;
         this.publishedProperties(false).filter(meta => ignore.indexOf(meta.id) < 0).forEach(meta => {
             if (this[meta.id + "_exists"]()) {
+                let value = this[meta.id]();
+                switch (meta.type) {
+                    case "widget":
+                        value = value.hashSum();
+                        break;
+                    case "widgetArray":
+                    case "PropertyArray":
+                        value = hashSum(value.map(v => v.hashSum()));
```

Comment:
It might be a good idea to rename the imported `hashSum` to `utilityHashSum` to avoid confusion?

Reply:
`value` seems to be unused in this context? Does assigning the response from `hashSum` to it somehow affect `this[meta.id]` as a pass by reference?

Reply:
It is used further down the source code (just not in the changed code).

Reply:
Hmmm it does look iffy.